README.md

```diff
@@ -48,3 +48,40 @@ Then browse to http://localhost:8090/github to see the public UI.
 [OCurrent]: https://github.com/ocurrent/ocurrent
 [OCluster]: https://github.com/ocurrent/ocluster
 [opam-repository]: https://github.com/ocaml/opam-repository
+
+
+### Local development
```

Comment:
It's a bit confusing to have a "Local development" section that differs from the local development setup described earlier in the file (which uses `opam-repo-ci-local` for a simpler but less complete approximation of the real system).